doc/src/install_guide/installation_guide.md

```diff
@@ -11,202 +11,149 @@
-is that the other users do not need to have the build toolchain on their
-systems to install the OPAE SDK.
-
-* To build rpm packages follow these steps:
 
+When installed finished, reboot your system.
+When the system login again, check the kernel version is correct.
 ```console
-$ cd opae-sdk-<release>
-$ mkdir mybuild
-$ cd mybuild
-$ cmake .. -DOPAE_BUILD_SIM=ON -DCPACK_GENERATOR=RPM -DCMAKE_INSTALL_PREFIX=<desired install location>
-$ make package_rpm
+[figo@localhost linux-dfl]$ uname -a
+Linux localhost.localdomain 5.8.0-rc1-dfl-g73e16386cda0 #6 SMP Wed Aug 19 08:38:32 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux
 ```
-.. note::
-```
-Note: Providing CMAKE_INSTALL_PREFIX is optional, by default the install prefix will be /usr.
-```
-This will generate the following rpm packages.
 
+And also you can check the OPAE dfl drivers have auto-loaded or not.
 ```console
-opae-<release>.x86_64.rpm               (meta package)
-opae-libs-<release>.x86_64.rpm          (libopae-c and samples)
-opae-tools-<release>.x86_64.rpm         (base tools)
-opae-tools-extra-<release>.x86_64.rpm   (extra tools)
-opae-devel-<release>.x86_64.rpm         (headers)
-opae-ase-<release>.x86_64.rpm           (libopae-c-ase)
+[figo@localhost linux-dfl]$ lsmod | grep fpga
+ifpga_sec_mgr          20480  1 intel_m10_bmc_secure
+fpga_region            20480  3 dfl_fme_region,dfl_fme,dfl
+fpga_bridge            24576  4 dfl_fme_region,fpga_region,dfl_fme,dfl_fme_br
+fpga_mgr               16384  4 dfl_fme_region,fpga_region,dfl_fme_mgr,dfl_fme
+[figo@localhost linux-dfl]$ lsmod | grep dfl
+dfl_eth_group          36864  0
+dfl_fme_region         20480  0
+dfl_emif               16384  0
+dfl_n3000_nios         20480  0
+dfl_fme_br             16384  0
+dfl_fme_mgr            20480  1
+dfl_fme                49152  0
+dfl_afu                36864  0
+dfl_pci                20480  0
+dfl                    40960  7 dfl_pci,dfl_fme,dfl_fme_br,dfl_eth_group,dfl_n3000_nios,dfl_afu,dfl_emif
+fpga_region            20480  3 dfl_fme_region,dfl_fme,dfl
+fpga_bridge            24576  4 dfl_fme_region,fpga_region,dfl_fme,dfl_fme_br
+fpga_mgr               16384  4 dfl_fme_region,fpga_region,dfl_fme_mgr,dfl_fme
 ```
 
-* To build deb packages follow these steps:
-
- .. note::
-  ```
-  Note: For generating deb packages, cmake version 3.0.0 and above is required.
-  ```
+## Build the OPAE-SDK ##
+Before you build the SDK, you must install the required packages. Run the following commands:
 
 ```console
-$ cd opae-sdk-<release>
-$ mkdir mybuild
-$ cd mybuild
-$ cmake .. -DOPAE_BUILD_SIM=ON -DCPACK_GENERATOR=DEB -DCMAKE_INSTALL_PREFIX=<desired install location>
-$ make package_deb
-```
-.. note::
-```
-Note: Providing CMAKE_INSTALL_PREFIX is optional, by default the install prefix will be /usr.
+$ sudo yum install cmake libuuid libuuid-devel json-c sphinx libtool python3-devel python3-libs json-c-devel hwloc-devel uuid python3-pip python3-virtualenv tbb-devel rpm-build
```

Comment:
Is libtool required?
Is rpm-build required?
Is sphinx required?

Reply:
libtool and sphinx, i have no check, but i saw it is need on our VC production release.
the rpm-build is need, because we need build the rpm.

Reply:
i  remove the libtool and sphinx，and it can compile fine.